libr/anal/rtti_msvc.c

```diff
@@ -387,27 +387,20 @@ R_API char *r_anal_rtti_msvc_demangle_class_name(const char *name) {
 		|| strncmp (name + original_len - 2, "@@", 2) != 0) {
 		return NULL;
 	}
-	char *ret = malloc ((original_len - 6) * 2 + 1);
-	if (!ret) {
-		return NULL;
-	}
-	char *c = ret;
-	const char *oc = name + original_len - 3;
-	size_t part_len = 0;
-	while (oc >= name + 4) {
-		if (*oc == '@') {
-			memcpy (c, oc + 1, part_len);
-			c += part_len;
-			*c++ = ':';
-			*c++ = ':';
-			part_len = 0;
+	char *ret = context->anal->coreb.cmdstrf (context->anal->coreb.core, "\"iD msvc %s\"", name);
```

Comment:
This can lead to r2 injection because name is untrusted input from the binary. I think it would be better to call the API from RBin directly, if necessary through RBinBind. (might need a bit of refactoring on RBin side)